spec/api_client.js

```diff
@@ -75,45 +75,45 @@ describe('api client', function () {
     apiClient = new ApiClient(urlBase, serviceId, apiKeyId);
     apiClient.post(path, data)
       .then(function (response) {
-        expect(response.statusCode).to.equal(200);
+        expect(response.status).to.equal(200);
         done();
     });
   });
 
   it('should direct get requests through the proxy when set', function (done) {
     var urlBase = 'https://api.notifications.service.gov.uk',
-      proxyUrl = 'http://proxy.service.gov.uk:3030/',
+      proxyConfig = { host: 'addressofmyproxy.test'},
```

Comment:
This, and the other uses of `proxyConfig`, seems to be missing the port and does the host need to change?

Reply:
So I haven't replicated the test exactly. If I wanted to, and there is no reason I shouldn't, I would instead need

```proxyConfig = { host: 'proxy.service.gov.uk', port: 3030 }```

However it appears that port is optional in the same way that `auth` is:

![image](https://user-images.githubusercontent.com/7228605/92120422-f2395900-edf0-11ea-9f17-5a9a09670742.png)

So I think the test is still valid written as is, but is a little different in terms of the url that was being tested last time for no good reason than me changing it whilst developing.